main/main.go

```diff
@@ -167,7 +174,12 @@ func main() {
 	}
 
 	// Initialize the database
-	db, err := database.NewJSONFileStorage(logger, cfg.StorageLocation, sourceBlockchainIDs)
+	sourceChainIDs := make([]ids.ID, len(cfg.SourceBlockchains))
```

Comment:
Same request here (`sourceBlockchainIDs`)

Reply:
Actually, could we just pass `cfg.SourceBlockchains` directly to `NewJSONFileStorage`? I don't think it's necessary to do an iteration to extract the blockchain ID values, just to then iterate over the new slice.